util/typeshelper/typeshelper_test.go

```
@@ -0,0 +1,42 @@
+package typeshelper
```

golangci-lint (1.62.2)

Missed header for check

(goheader)